File: src/main/java/de/markerud/upgrade/Application.java
package de.markerud.upgrade;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class Application {

    public static void main(String[] args) {
        System.setProperty("reactor.netty.http.server.accessLogEnabled", "true");
        SpringApplication.run(Application.class, args);
    }

}

File: src/main/kotlin/de/markerud/upgrade/Application.kt
package de.markerud.upgrade

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.zalando.logbook.autoconfigure.webflux.LogbookWebFluxAutoConfiguration

@SpringBootApplication(exclude = [LogbookWebFluxAutoConfiguration::class])
class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}
